.github/workflows/docs-netlify.yaml

```diff
@@ -0,0 +1,37 @@
+name: Publish docs to Netlify 
+
+on:
+  release:
+    types: [published] # remove auto deploying docs everytime pull request is merged
+  workflow_dispatch:
+
+permissions:
+  contents: write 
+
+jobs:
+  publish-docs:
+    name: Publish docs to Netlify
+    if: ${{ !github.event.release.prerelease }}
```

Comment:
Will this work when manually running the workflow?

Reply:
yeah, it should work.